receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -191,17 +193,75 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 			intraRequestCache[hashedLabels] = rm
 		}
 
+		scopeName, scopeVersion := prw.extractScopeInfo(ls)
+		metricName := ls.Get(labels.MetricName)
+		unit := ""
+		// TODO: Like UnitRef, we should assign the HelpRef to the metric.
+		if ts.Metadata.UnitRef > 0 && ts.Metadata.UnitRef < uint32(len(req.Symbols)) {
+			unit = req.Symbols[ts.Metadata.UnitRef]
+		}
+
+		// Temporary approach to generate the metric key.
+		// TODO: Replace this with a proper hashing function.
+		metricKey := fmt.Sprintf("%d:%s:%s:%s:%s:%d",
+			hashedLabels,     // Resource identity
```

Comment:
Let's use the `identity` package here to create a hash of the resource

Reply:
addressed